config/samples/resources/artifactregistryrepository/artifactregistry_v1beta1_artifactregistryrepository.yaml

```diff
@@ -18,6 +18,9 @@ metadata:
   name: artifactregistryrepository-sample
   labels:
     label-one: "value-one"
+  annotations:
```

Comment:
Just FYI these samples do make it into our docs (and tests), so we have to be a little careful.  That said this is done in a bunch of other places so should be fine.  _Maybe_ we shouldn't be referencing the sample here, but I'm not sure I feel too strongly about that.